### ISSUE TITLE
Translations: Chinese time order

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1624]

**Comment by greg-does-weather** [comment]
To the extent practicable, we should not translate dates. Instead, we should use browsers' built-in [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) API to format from a standard ISO string into the browser's locale.

**Comment by colinmurphy01** [comment]
Good point. If date and time are not embedded in the text fields of the data we return, then hopefully that should be sufficient and this isn't something we need to solve.

**Comment by colinmurphy01** [comment]
Might not be able to deliver by end of sprint.
